### PR TITLE
feat(UserAvatar): add `srcset` support

### PR DESCRIPTION
**Pull request by ogonkov**

### CODE REVIEW

File: src/components/index.ts [hunk 1]
```diff
@@ -67,3 +67,5 @@ export * from './utils/setRef';
 export {useOnFocusOutside} from './utils/useOnFocusOutside';
 export * from './utils/xpath';
 export * from './utils/useFileInput/useFileInput';
+export * from './utils/getSrcSet';
+export * from './utils/getClosestNumber';
```
Comment (amje) [inline src/components/index.ts:71]: Why we export these two?

File: src/components/UserAvatar/UserAvatar.tsx [hunk 2]
```diff
@@ -1,28 +1,58 @@
...
+                    srcSet={srcSet}
+                    sizes={sizes}
+                    alt={''}
+                />
+            </div>
+        );
+    }
+
     return (
         <div
```
Comment (amje) [inline src/components/UserAvatar/UserAvatar.tsx:51]: I think we should always use  in the component

File: src/components/index.ts (hunk 1 above)
Comment (ogonkov) [reply src/components/index.ts:71]: I guess they could be omited, will remove this exports

File: src/components/utils/getClosestNumber/__tests__/getClosestNumber.test.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import {getClosestNumber} from '../getClosestNumber';
+
+test('should find appropriate number', () => {
+    expect(getClosestNumber(1, [1, 2, 3, 50])).toBe(1);
+    expect(getClosestNumber(20, [1, 2, 15, 30, 50])).toBe(30);
```
[the remark this answers is not in the document]
Comment (ogonkov) [reply src/components/utils/getClosestNumber/__tests__/getClosestNumber.test.ts:5]: Yeah, just next number that is bigger than given or equal to it

**Review by amje — APPROVED** [review]

**Pull request merged**